src/shared/components/heroBanner/heroBanner.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './heroBanner.css';
+
+const propTypes = {
+  imageSrc: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
+  className: PropTypes.string.isRequired,
+  subtitle: PropTypes.string,
+  isLeftFocus: PropTypes.bool,
+  isCenterFocus: PropTypes.bool,
+  isRightFocus: PropTypes.bool
```

Comment:
@ksmacleod99 If you're wanting to hold off on using the properties `isLeftFocus`, `isRightFocus`, and `isCenterFocus', best to just remove those properties here (they're not being used).

If you're trying to set different properties on something, you could have a corresponding CSS class for each of those boolean properties, inside `./heroBanner.css`.  Then, inside `heroBanner.js` you could use some JS logic to add the correct class to the JSX.